grid/customer-management/create-customer/maps/shopify.suma.js

```diff
@@ -0,0 +1,65 @@
+function CreateCustomer({ input, parameters, services }) {
```

Comment:
nitpick: Ideally this function is the entry - there is no need for indirection to call `createCustomer` since its body can just be directly in this function.

Reply:
Yep, this is on the todo list :)